### PR TITLE
Fix for goal clerk sign to a file

### PR DESCRIPTION
**Pull request by pzbitskiy**
## Changes

Restore ability to save transaction to a file that was missed
during LogicSig and Assets implementation

## Test Plan

Manual test and goal cmd tests for Jenkins later

### CODE REVIEW

File: cmd/goal/clerk.go [hunk 1]
```diff
@@ -344,18 +349,14 @@ var sendCmd = &cobra.Command{
...
 			}
-		} else {
-			stx = transactions.SignedTxn{
-				Txn: payment,
-			}
-			stx = populateBlankMultisig(client, dataDir, walletName, stx)
 		}
+
 		if txFilename == "" {
 			// Sign and broadcast the tx
```
Comment (algobolson) [inline cmd/goal/clerk.go]: update this comment to reflect that the tx will have already been signed

Comment (pzbitskiy) [reply cmd/goal/clerk.go]: done

**Comment by pzbitskiy** [comment]
@tsachiherman already did - see https://github.com/algorand/go-algorand-internal/issues/187

**Comment by tsachiherman** [comment]
Thanks - btw, I forget to congratulate you on an excellent find. ;-)

**Pull request merged**